vue/components/ui/organisms/header/announcement-modal.vue

```diff
@@ -0,0 +1,281 @@
+                        v-if="reaction"
+                        v-bind:id="`${index}`"
+                        v-on:click="onReactionClick"
+                    />
+                </div>
+            </div>
+        </div>
+    </transition>
+</template>
+
+<style lang="scss" scoped>
+@import "css/colors.scss";
+@import "css/variables.scss";
+@import "css/animations.scss";
+
+.announcements-container {
+    animation: fade-grow-rise 0.35s cubic-bezier(0.645, 0.045, 0.355, 1);
+    background-color: $white;
+    border: 1px solid transparent;
+    width: 370px;
+}
+
+.announcements-container.fade-leave-active {
+    animation: fade-shrink-visibility 0.25s cubic-bezier(0.645, 0.045, 0.355, 1) forwards;
+}
+
+.announcements-container.visible {
+    border: 1px solid $light-white;
+    box-shadow: 0px 6px 24px 0px rgba(67, 86, 100, 0.15);
+}
+
+.announcements-container .announcement-header {
+    border-bottom: 1px solid $light-white;
+    padding: 15px 24px 15px 24px;
+}
+
+.announcements-container .announcement-header .title {
+    display: inline-block;
+    font-size: 18px;
+    line-height: 18px;
+}
+
+.announcements-container .announcement-header .description {
+    font-size: 14px;
+    line-height: 14px;
+    margin: 0px 0px 10px 0px;
+}
+
+.announcements-container .announcement-header ::v-deep .checkbox .choice {
+    padding-left: 0px;
+}
+
+.announcements-container .announcement-header ::v-deep .button-icon {
+    display: inline-block;
+    float: right;
+    margin-top: 16px;
+}
+
+.announcements-container .announcements {
+    max-height: 900px;
+    overflow-y: auto;
+}
+
+.announcements-container .announcements .announcement {
+    border-top: 1px solid $light-white;
+    padding: 0px 24px 15px 24px;
+    position: relative;
+}
+
+.announcements-container .announcements .announcement:first-child {
+    border-top: 0px;
+}
+
+.announcements-container .announcements .announcement .date {
+    display: inline-block;
+    line-height: 14px;
+    margin-top: 5px;
+}
+
+.announcements-container .announcements .announcement .dot {
+    background-color: #4b8dd7;
+    border: 1px solid #ffffff;
+    border-radius: 50%;
+    display: inline-block;
+    float: right;
+    height: 8px;
+    line-height: 14px;
+    padding: 0px 0px 0px 0px;
+    position: absolute;
+    right: 30px;
+    top: 27px;
+    width: 8px;
+}
+
+.announcements-container .announcements .announcement .dot.clickable {
+    cursor: pointer;
+}
+
+.announcements-container .announcements .announcement .title {
+    font-size: 16px;
+    line-height: 22px;
+    margin: 0px 0px 0px 0px;
+}
+
+.announcements-container .announcements .announcement .content {
+    font-size: 14px;
+    line-height: 25px;
+    margin-bottom: 40px;
+    white-space: pre-line;
+}
+
+.announcements-container .announcements .announcement .link {
+    border-color: #1d2631;
+}
+
+.announcements-container .announcements .announcement .reaction {
+    float: right;
+    line-height: initial;

```

Comment:
Please update to the newest reaction and confirm if this is needed.

Reply:
Fix in ![PR](https://github.com/ripe-tech/ripe-components-vue/pull/108)